src/55functions.js

```diff
@@ -157,7 +157,17 @@ stdlib.LOWER = stdlib.LCASE = function(s) {return und(s,'String(y).toLowerCase()
 //stdlib.LCASE = function(s) {return '('+s+').toLowerCase()';}
 
 
-// LTRIM
+// Returns a character expression after it removes leading blanks.
+// see https://docs.microsoft.com/en-us/sql/t-sql/functions/ltrim-transact-sql
+stdlib.LTRIM = function(s) {
+    return und(s, 'y.substr(y.indexOf(y.trim()), y.length)');

```

Comment:
it took me a little time to find out what is going on. 

I realise that you are doing a full trim to find the final string to identify the how much to remove. There seem to be some performance issues when dealing with long strings. Also, it feels a bit off to trim both ends to be able to trim one end. 

I suggest you change to a solution based on regular expression. I believe something like the following should do the trick. 

``` 
y.replace(/^\s+/,"");
```